sda/cmd/auth/main.go

```diff
@@ -118,13 +118,16 @@ func (auth AuthHandler) postEGA(ctx iris.Context) {
 
 	res, err := authenticateWithCEGA(auth.Config.Cega, username)
 
+	var statusCode int
 	if err != nil {
 		log.Error(err)
+		statusCode = 404
+	} else {
+		statusCode = res.StatusCode
+		defer res.Body.Close()
```

Comment:
```suggestion
	if err != nil {
		log.Errorf("cega connection error: %v",err)
		res = &http.Response{
			Body: io.NopCloser(nil),
			StatusCode: http.StatusInternalServerError,
		}
```

This also means that the 404 case below needs to be replaced with:
`case 500,502,503:` so that we cover all bases when the NSS server misbehaves.